index.html

```diff
@@ -705,6 +705,18 @@ <h3>Content Distribution Networks</h3>
       </p>
     </section>
 
+<section class="informative">
+<h3>Malicious Issuers</h3>
+<p>
+In general, this specification cannot protect data <a>subject</a> from tracking by a malicious <a>issuer</a>.
+</p>
+<p>
+A malicious <a>issuer</a> might intentionally attack herd privacy by creating a unique status list
+per credential issued in order to establish a 1-1 mapping to track when a <a>verifier</a> processes a specific credential.
+</p>
```

Comment:
```suggestion
<p>
A malicious <a>issuer</a> might intentionally attack herd privacy by creating a
unique status list per credential issued in order to establish a 1-1 mapping to track
when a <a>verifier</a> processes a specific credential. Similarly, they could establish
another a 1-1 mapping by using a different cryptographic key for every credential
issued that is tracked in a status list.
</p>
```

Reply:
Notably, if the holder fetches the status list on their own and passes that to the verifier, they could perhaps avoid this attack, but it's not really avoidable (or less so) if the issuer uses a different key for each credential issued.